cli/src/main/java/com/devonfw/tools/ide/tool/GlobalToolCommandlet.java

```diff
@@ -70,6 +70,15 @@ protected boolean runWithPackageManager(boolean silent, List<PackageManagerComma
     return false; // None of the package manager commands were successful
   }
 
+  private void logPackageManagerCommands(PackageManagerCommand pmCommand) {
+
+    this.context.info("We need to run the following privileged command(s):");
+    for (String command : pmCommand.commands()) {
+      this.context.info(command);
+    }
+    this.context.info("This will require root permissions!");
```

Comment:
Sorry for interfering once more but I just read my own story:
> Then runWithPackageManager should log an according message on log-level interaction.

So please change the loglevel from INFO to INTERACTION.

Reply:
Thanks for rework 👍